### DISCUSSION TITLE
Enterprise account needed for SSH for an organisation?

### DISCUSSION THREAD

**Discussion by mprofitlich** [community/community#34333] [Enterprise]
I am trying to connect my Hosting to Github via SSH. For that I wanted to create a key at Settings › Authentification security. 

Is it true that I need an enterprise account for using SSH for my organisation? Or is it possible using free or at least team accounts?

Thank you
Moritz

**Comment by dgholz** [comment]
> Is it true that I need an enterprise account for using SSH for my organisation?

No.

> Or is it possible using free or at least team accounts?

Yes. You can add an SSH key to an account, and use it to interact with repositories in free, Team, or Enterprise organisations. Though probably what you need here is [a deploy key on the repository](https://docs.github.com/en/developers/overview/managing-deploy-keys#deploy-keys).

**Reply by mprofitlich** [reply]
Thank you very much! I created a deploy key – but in the clone dropdown for SSH Github still says: You don't have any public SSH keys in your GitHub account. You can add a new public key, or try cloning this repository via HTTPS.

Can I ignore hat message? I managed to clone the repository to the webserver via SSH shell but when using the Hoster’s Plesk Git assistant I get an error message that I have no access rights.

<img width="495" alt="grafik" src="https://user-images.githubusercontent.com/6146601/192569532-dc713a4f-bc73-4af5-b86f-fe45b7cbb2df.png">
